game/scripts/npc/abilities/morphling_adaptive_strike_agi.txt

```diff
@@ -24,7 +24,7 @@
 
     // Time
     //-------------------------------------------------------------------------------------------------------------
-    "AbilityCooldown"                                     "10"
+    "AbilityCooldown"                                     "8" //OAA
```

Comment:
Morphling can be good in the right hands but with oaa there's so much going on you'd have to be Nisha to pull it off. I don't want to buff the damage or stun on either adaptive strikes as I don't see anything inherently wrong with the hero he just doesn't fit in oaa compared to other heroes. a small CD buff with both his spells as well as a shared CD will allow more flexible gameplay from Morph player, whom desperately need any second they can get to survive in the world of OAA.

Reply:
I hate morphling so I can't say much on this. This change seems meaningless. This is not the issue of the hero.

Reply:
from my experience the spell is a bad linkens popper. the time it takes to cast is probably less damage efficient than just right clicking. the type or morph that gets any use out of adaptive strike has serious problems that wouldn't be answered by this. There's no real problem with adding this but it seems kind of irrelevant

Reply:
> from my experience the spell is a bad linkens popper. the time it takes to cast is probably less damage efficient than just right clicking. the type or morph that gets any use out of adaptive strike has serious problems that wouldn't be answered by this. There's no real problem with adding this but it seems kind of irrelevant

Right now if you play morph you have to either go full-in agi glasscannon, which can be countered by 1 or 2 items and the natural scaling of the game itself, or full tank and do pretty much no dmg and have a super bad abilities for that role. Morph only excels if he snowballs super early. I want to shift these playstyles into more of a middling playstyle, utilizing both agi and strength to succeed. Adaptive strike changes area the first of my ideas to incorporate and if he needs more changes afterwards than we'll do so, but I don't think we'll need it.

Reply:
Revert all morphling changes in this PR and make a seperate PR for morphling. And then consult other morphling players how to buff a broken hero. I don't want to be a part of this.